src/database/db_post.js

```diff
@@ -0,0 +1,44 @@
+const connect = require('./db_connect');
```

Comment:
With the file name `db_post`, I am not convinced grouping queries by http method is the best way.

Rather than
`post.memory()`, `post.user()`, `get.memory()` ...

I would structure stuff like
`memory.create(data)`, `memory.get(memoryId)`

In terms of the verbs:

HTTP METHOD | function name
---|---
GET | get
POST | create
PUT | Update
DELETE | delete

Does that make sense?